dev/com.ibm.ws.jbatch.misc_fat/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+	util/src,\
+	test-applications/implicit/src,\
+	test-applications/implicit/resources
+
+fat.project: true
+
+# Not 100% sure what this means, just put it in to have the test pass.
+tested.features: \
+    cdi-2.0,\
+    servlet-4.0
+
```

Comment:
Hmm..I wonder why batch-1.0 doesn't need to be there.

Reply:
We will automatically scan server.xml files for tested features at build time.  For cases where the test dynamically alters the feature list at runtime though, the dynamically added features need to be specified here.  Since you are using the reapeat with EE8 features `@ClassRule` you need to specify the EE8 equivalents here.